src/JsRoot/hook/useJsRootCanvas.tsx

```diff
@@ -1,16 +1,16 @@
 import { redraw, resize } from 'jsroot';
 import { forwardRef, useCallback, useEffect, useMemo, useRef, useState } from 'react';
 import { useVisible } from 'react-hooks-visible';
-import { mergeRefs } from 'react-merge-refs';
```

Comment:
That seems the only place where `mergeRefs` was used. If its not used anymore, then maybe we can remove if from list of dependencies in `package.json` ?